handbook/communication/rfcs/index.md

```diff
@@ -1,139 +1,173 @@
-# Requests for comments (RFCs)
+# Requests for Comments (RFCs)
 
-We value writing down plans so that we can asynchronously communicate to and solicit feedback from our [all-remote](../../company/remote/index.md) team. A good plan communicates what problem is being solved, why that problem is being prioritized now, who will make the final decision(s), and what the plan is to solve the identified problem.
-
-This document describes how we operationalize written planning through our RFC process. RFC literally means "Request for Comments" and you can think about it as exactly that, no more, no less.
-
-This process is designed to be lightweight so that it can be used for many purposes (e.g. product specs, policy decisions, technical discussion), and it is optimized for facilitating collaboration and feedback. In contrast, GitHub issues are best for tracking concrete bug reports or work that has already been scoped and planned (i.e. there isn't much remaining to discuss).
-
-We encourage you to understand [how decisions are made at Sourcegraph](../decisions.md) before writing an RFC.
+We use RFCs to collaborate successfully as an [all-remote](../../company/remote/index.md) team.
 
-<i>All public RFCs are in a [public Google Drive folder](https://drive.google.com/drive/folders/1zP3FxdDlcSQGC1qvM9lHZRaHH4I9Jwwa). Some RFCs contain non-public information, and are in a [private Google Drive folder](https://drive.google.com/drive/folders/1KCq4tMLnVlC0a1rwGuU5OSCw6mdDxLuv). For examples of public vs. non-public information, see our [transparency chart](https://about.sourcegraph.com/handbook/company/values#open-and-transparent).</i>
+RFCs give us a way to write down ideas and plans so we can communicate, collect thoughtful feedback from others on the team, and [make decisions](../decisions.md).
 
-## Status
+An “RFC” literally means a “Request for Comments.” Though RFCs at Sourcegraph can take many forms for different purposes, an RFC is ultimately just a document with a few attributes:
 
-Each RFC has a status that is in the title of the RFC (e.g. "RFC 1 WIP: Title"). The author is responsible for keeping the status updated.
+- The author’s name(s)
+- The date it was written
+- A sequential number
+- A status label
```

Comment:
I am wondering if we should bias more towards status prose in the doc rather than labels in the title.

Reply:
I think there is a lot of value to be had from having the status in the title. Since we don't use a structured format for our RFC (like Markdown documents in a repo) it's not easy to extract metadata like status in tooling. Take for example the value you get when using `sg rfc list` by having it in the title.
<img width="1147" alt="screen-capture--2021-09-28--0918@2x" src="https://user-images.githubusercontent.com/492942/135041583-8d6e54a3-223d-46e6-a250-f8147716472a.png">

Reply:
Based on this feedback, let's keep the status in the title until we have some other solution to exposing better metadata.